app/controllers/concerns/auth.rb

```diff
@@ -18,18 +18,6 @@ module Auth
     end
   end
 
-  def current_user
```

Comment:
We can also remove the `class UserNotAuthorised` in this file (GitHub doesn't let me point at it directly) which is now no longer in use